bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -178,24 +178,42 @@ export class PlotCanvasView extends BokehView
     # Update any DataRange1ds here
     frame = @model.frame
     bounds = {}
+    log_bounds = {}
+
+    calculate_log_bounds = false
+    for r in _.values(frame.x_ranges).concat(_.values(frame.y_ranges))
+      if r instanceof DataRange1d
+        if r._mapper_type == "log"
```

Comment:
Ok, if I'm going to make this property public, then do I use `@define`? I'm not entirely clear on when `@define`, `@internal`, etc. are supposed to be used.